src/app/Models/Setting.php

```diff
@@ -4,11 +4,34 @@
 
 use Backpack\CRUD\CrudTrait;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Storage;
+use Intervention\Image\Facades\Image;
 
 class Setting extends Model
 {
     use CrudTrait;
 
     protected $table = 'settings';
     protected $fillable = ['value'];
+
+
+    public function setValueAttribute($value)
+    {
+        $attribute_name = "value";
+        $disk = "public";
+        $destination_path = $this->key;
+        if ($value==null) {
+            Storage::disk($disk)->delete($this->image);
```

Comment:
Should one/someone check the return of this? I forget if a `Storage::delete` throws an exception of not if a delete fails.